### DISCUSSION TITLE
Followed "Getting started", getting errors

### DISCUSSION THREAD

**Discussion by paulmoloneyr3** [thuliteio/doks#1259] [Q&A] [answered]
I deployed a blank instance of Doks as per the instructions here:

https://getdoks.org/docs/start-here/installation/

However, when I run npm run dev, I get the following multiple errors:

https://pastebin.com/Kqd3CGvR 

Any help appreciated,

P.

**Comment by h-enk** [comment, marked as answer]
What Hugo/Node version are you using (use the latest)?

Working setup:
- Node v20.11.0
- npm 10.2.4
- Hugo extended  v0.129.0

**Reply by paulmoloneyr3** [reply]
Thanks, updating everything worked.
